py/torch_tensorrt/dynamo/lowering/_remove_sym_nodes.py

```diff
@@ -10,17 +10,18 @@ def remove_sym_nodes(gm: torch.fx.GraphModule) -> torch.fx.GraphModule:
     dynamic=True behavior
     """
     # Extract SymInt placeholder Tensors
-    placeholders = [
+    placeholder_sym_ints = [
         node
         for node in gm.graph.nodes
         if (
             node.op == "placeholder"
             and isinstance(node.type, type)
             and issubclass(node.type, torch.SymInt)
+            and not node.users
```

Comment:
Do `SymInt` inputs always have 0 users?

Reply:
Nope they have users. But there's one no-user sym_int node added by torch.compile workflow when we do `torch._dynamo.mark_dynamic` and this is removing that specific node.